From: Jehan <jehan@girinstud.io>
Date: Tue, 24 Dec 2019 01:32:57 +0100
Subject: Fix invalid property name in pagecurl plugin

Issue #4392: Gimp Segmentation Fault triggered by Glib GParamSpec property
name validation.

Previous commit fixed the parameter name validation in core code. This
commit fixes the source error in the plug-in code.

Origin: upstream, commit:573c043babce0620db715afa7696e5b06d7a0a3e
Applied-upstream: 2.10.16
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=953880
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=953854
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=953794
---
 plug-ins/pagecurl/pagecurl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plug-ins/pagecurl/pagecurl.c b/plug-ins/pagecurl/pagecurl.c
index 048395e..c82dc8a 100644
--- a/plug-ins/pagecurl/pagecurl.c
+++ b/plug-ins/pagecurl/pagecurl.c
@@ -195,7 +195,7 @@ query (void)
 
   static const GimpParamDef return_vals[] =
   {
-    { GIMP_PDB_LAYER, "Curl Layer", "The new layer with the curl." }
+    { GIMP_PDB_LAYER, "curl-layer", "The new layer with the curl." }
   };
 
   gimp_install_procedure (PLUG_IN_PROC,
